### PR TITLE
Always use global login & add success message for password change

### LINKED ISSUE #1039: After signing out, a user should be redirected to the "global" login-mask

### LINKED ISSUE #1040: After changing the own password a user should get a success message
They should see a "successful" message (by a snackbar).

### PR DESCRIPTION
**Pull request by jsangmeister**
fixes #1039 
fixes #1040

- Remove meeting-specific login and always use the global one
- Add success message by snackbar after password change (translation still missing)
- Add QoL changes:
  - shift+enter can now also save in the user detail & password change modal
  - "show/hide password" buttons are no longer focusable and are not clicked by pressing enter

### CODE REVIEW

**Review by emanuelschuetze — APPROVED** [review]
successfully tested

**Review by GabrielInTheWorld — APPROVED** [review]

File: client/src/app/ui/modules/global-headbar/modules/account/components/account-dialog/account-dialog.component.ts [hunk 1]
```diff
@@ -144,7 +165,6 @@ export class AccountDialogComponent extends BaseUiComponent implements OnInit {
     }
 
     private async updateIsUserInScope(): Promise<void> {
-        this._isUserInScope = true;
-        // this._isUserInScope = await this.repo.isUserInSameScope(this.operator.operatorId);
+        this._isUserInScope = await this.userService.isUserInSameScope(this.operator.operatorId);
```
Comment (jsangmeister) [inline client/src/app/ui/modules/global-headbar/modules/account/components/account-dialog/account-dialog.component.ts:168]: This seemed like an accidental regression @GabrielInTheWorld - please correct me if I'm wrong

**Comment by jsangmeister** [comment]
removed duplicate `password-form`

**Review by GabrielInTheWorld — APPROVED** [review]

**Comment by GabrielInTheWorld** [comment]
Please resolve the conflict and then you can merge.

**Pull request merged**